### ISSUE TITLE
Separation between buttons in checkboxGroupButtons

### ISSUE THREAD

**Issue by dbarg1** [dreamRs/shinyWidgets#233]
Hi, 

Thanks so much for the package - I've found it very useful.

I'm wondering - is it possible to have a gap (whitespace) between the buttons in checkboxGroupButtons ? Visually, it would be very helpful. I've tried adjusting all of the associated options, but not had much luck.

Thanks in advance.

**Comment by pvictor** [comment]
Hello,
Arer you looking for `individual = TRUE` ?

![image](https://user-images.githubusercontent.com/4415580/67939401-c1d1ce80-fbd1-11e9-8c6a-c189c4d70394.png)

```r
library(shiny)
library(shinyWidgets)

ui <- fluidPage(
  tags$h1("checkboxGroupButtons examples"),
  
  checkboxGroupButtons(
    inputId = "somevalue",
    label = "Make a choice: ",
    choices = c("A", "B", "C"), 
    individual = TRUE
  ),
  verbatimTextOutput("value")
)
server <- function(input, output) {
  
  output$value <- renderPrint({ input$somevalue })
  
}
shinyApp(ui, server)
```

Victor

**Comment by dbarg1** [comment]
Hi - thanks for the reply! I've tried the "individual = TRUE" option. My issue with it is that I want there to more separation between buttons. Ideally, I want to be able to increase the size of the buttons, and maintain gaps between them. When I set "justified = TRUE", the gap between them disappears. Is it possible to have the checkbox buttons be the same width, and justified with a slight offset (gap) between them? As in the attached picture. Is this do-able using shinyWidgets?

Thanks in advance.

<img width="899" alt="checkbox_buttons" src="https://user-images.githubusercontent.com/5533513/67981529-3c611500-fbf7-11e9-9383-1e38a7038a5b.png">
